### PR TITLE
feat: Update User Interface for opening and closing a review

### LINKED ISSUE #1328: Handle Submission when Accepted for Review
### Problem/Why

The interactive elements on the Submissions page and Submission Review page currently do not change when a submission is accepted for review. They show the same interactions that are available when the submission is initially submitted, which is an unideal user experience.

### Goal/What

An open review button should appear in place of the fitness for review interactions on the submission review page. Once the submission is opened for review, interactions need to appear to enable the reviewer/review coordinator to close the review. 

### Solution/How

Replace interactions for (accept, request resubmit, reject) with an button that enables the review/review coordinator to open the review.

After the review is opened, replace the open review button with a close review button.

### To Do / Acceptance Criteria

- [x]  Interactions to determine fitness for review are replaced with an open review button once a submission status is changed from initially submitted to awaiting review
- [x] Open review button is replaced with a close review button after the review is opened
- [x] Add Cypress testing to ensure interactions work as expected

### Review Checklist

- [x] An editor can open a submission for review
- [x] An editor can close a submission for review
- [x] An editor can view approval desk options (Accepted as Final, Reject, Request Resubmission)

### Accessibility Checklist

- [x] Open review button is keyboard accessible
- [x] All colors adhere to WCAG AA guidelines
- [x] The controls for opening, closing, and approval desk options are annouceable via screen reader

### PR DESCRIPTION
**Pull request by vilagshe**
Closes #1328 

Added open and close review buttons to the Submission Review page; changes decision options to reflect final decision actions.

### CODE REVIEW

File: client/src/components/dialogs/ConfirmStatusChangeDialog.vue [hunk 1]
```diff
@@ -102,6 +111,7 @@ async function updateStatus() {
       },
     })
   } catch (error) {
+    console.log(error)
```
Comment (gmeben) [inline client/src/components/dialogs/ConfirmStatusChangeDialog.vue:114]:
```suggestion
```
Remove this leftover console.log statement

File: client/src/pages/SubmissionsPage.vue [hunk 2]
```diff
@@ -121,6 +122,16 @@
                         "
                         >{{ $t("submission.action.accept_for_review") }}</q-item
                       >
+                      <q-item
+                        v-if="submission.status != 'INITIALLY_SUBMITTED'"
```
Comment (gmeben) [inline client/src/pages/SubmissionsPage.vue:126]: I'm noticing that this "Accept as Final" button as well as the other buttons still appear for a submission whose review has not yet been opened, which is inconsistent with the Submission Review page's display of the "Open for Review" button only. I think the available options should be consistent between the Submission Review page and the Submissions page.

A user on the Submissions page can choose options meant only for under review when a submission is out of review

**Review by gmeben — CHANGES_REQUESTED** [review]

File: client/src/pages/SubmissionsPage.vue (hunk 2 above)
Comment (vilagshe) [reply client/src/pages/SubmissionsPage.vue]: I think that makes sense to keep the options consistent. Made the suggested changes.

**Review by gmeben — APPROVED** [review]
Make sure to update from the master branch before merging into the [base branch](https://github.com/MESH-Research/CCR/tree/issue754-fitness-for-review).

**Pull request merged**